sui/its.js

```diff
@@ -19,48 +19,43 @@ function parseTrustedChains(config, trustedChain) {
 }
 
 async function setupTrustedAddress(keypair, client, config, contracts, args, options) {
-    const [trustedChain, trustedAddress] = args;
+    const [trustedChain] = args;
 
-    const { ITS: itsConfig } = contracts;
+    const { InterchainTokenService: itsConfig } = contracts;
 
-    const { OwnerCap, ITS } = itsConfig.objects;
+    const { OwnerCap, InterchainTokenService } = itsConfig.objects;
 
     const txBuilder = new TxBuilder(client);
 
     const trustedChains = parseTrustedChains(config, trustedChain);
 
-    const trustedAddressesObject = await txBuilder.moveCall({
-        target: `${itsConfig.address}::trusted_addresses::new`,
-        arguments: [trustedChains, trustedChains.map(() => trustedAddress)],
-    });
-
     await txBuilder.moveCall({
-        target: `${itsConfig.address}::its::set_trusted_addresses`,
-        arguments: [ITS, OwnerCap, trustedAddressesObject],
+        target: `${itsConfig.address}::interchain_token_service::add_trusted_chains`,
```

Comment:
check if it's already a trusted chain and throw an error. Otherwise, the tx will fail on simulation and it won't be clear to the user

Reply:
Again, cannot, you specifically asked to remove all getters not used by our contracts in Sui.

Reply:
https://github.com/axelarnetwork/axelar-contract-deployments/pull/517#discussion_r1939189806